ui/src/pages/error-pages/preLoginPage.jsx

```diff
@@ -1,23 +1,31 @@
 import React from "react";
-import { useAuth0 } from "@auth0/auth0-react";
-import { Button, Form, Nav, Card} from "react-bootstrap";
+import {useAuth0} from "@auth0/auth0-react";
+import {Button, Card, Nav} from "react-bootstrap";
 
 export default function PreLoginPage() {
   const {
     loginWithPopup,
   } = useAuth0();
 
-return <div className="challengesWrap">
+  return <div className="challengesWrap">
     <div className="container">
       <div className="row mt-30">
         <div className="col-lg-8 col-md-7 mainContent">
           <div className="grayCol">
-            <h2>How Wireless Technology Is Changing Business</h2>
-            <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical
-                Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at
-                Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a
-                Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the
-                undoubtable source.</p>
+            <h2>Welcome to Capture the Flag!</h2>
+
+            <p>Capture the flag is a web-based application that allows you to start learning the basics of
+              cyber
+              literacy. Here we have a collection of activities that you can interact with, whether its a
```

Comment:
Second to last word on the line, "its" should be "it's".

Reply:
Done